ckanext/data_qld/templates/scheming/snippets/display_field.html

```diff
@@ -30,6 +30,6 @@
   {%- if field.field_name.lower() == 'url' and field.label.lower() == 'source' -%}
     Not applicable
   {%- else -%}
-    Field name '{{field.field_name}}' not in data
+    [blank]
```

Comment:
There is still a scenario test expecting the old message syntax.

Reply:
not surprising. I'm unsure we we even need to be 'emitting'  that not in data message, it should just skip that row.